### PR TITLE
Balance Container redesigned according to Figma

### PR DESCRIPTION
**Pull request by RafiaChy**

### CODE REVIEW

File: lib/view_model/dashboard/balance_view_model.dart [hunk 1]
```diff
@@ -58,6 +58,19 @@ abstract class BalanceViewModelBase with Store {
...
+    @computed
+  String get asset {
+    if( appStore.wallet.currency == CryptoCurrency.btc){
+      return 'Bitcoin Assets';
+    }
+    else if(appStore.wallet.currency == CryptoCurrency.xmr){
+      return 'Monero Assets';
+    }
+    
+      return 'Litecoin Assets';
```
Comment (mkyq) [inline lib/view_model/dashboard/balance_view_model.dart:70]: Convert into switch, return empty string by default

File: lib/view_model/dashboard/balance_view_model.dart [hunk 2]
```diff
@@ -130,11 +142,10 @@ abstract class BalanceViewModelBase with Store {
       return '---';
     }
 
-    return fiatCurrency.toString() +
-        ' ' +
-        _getFiatBalance(
+    return   _getFiatBalance(
             price: price,
-            cryptoAmount: walletBalance.formattedAdditionalBalance);
+            cryptoAmount: walletBalance.formattedAdditionalBalance)+  ' ' +fiatCurrency.toString();
```
Comment (mkyq) [inline lib/view_model/dashboard/balance_view_model.dart:147]: add one space around "+" operator

File: lib/view_model/dashboard/balance_view_model.dart [hunk 3]
```diff
@@ -114,11 +127,10 @@ abstract class BalanceViewModelBase with Store {
       return '---';
     }
 
-    return fiatCurrency.toString() +
-        ' ' +
-        _getFiatBalance(
+    return  _getFiatBalance(
             price: price,
-            cryptoAmount: walletBalance.formattedAvailableBalance);
+            cryptoAmount: walletBalance.formattedAvailableBalance)+  ' ' +fiatCurrency.toString();
```
Comment (mkyq) [inline lib/view_model/dashboard/balance_view_model.dart:132]: add one space around "+" operator

File: lib/src/screens/dashboard/widgets/balance_page.dart [hunk 4]
```diff
@@ -8,7 +11,9 @@ class BalancePage extends StatelessWidget {
   BalancePage({@required this.dashboardViewModel});
 
   final DashboardViewModel dashboardViewModel;
-
+  ThemeBase get currentTheme => getIt.get<SettingsStore>().currentTheme;
+  Color get backgroundLightColor =>
+      currentTheme.type == ThemeType.bright ? Colors.transparent : Colors.white;
```
Comment (mkyq) [inline lib/src/screens/dashboard/widgets/balance_page.dart:16]: Add SettingsStore as property to BalancePage class and pass SettingsStore instance through constructor and inject it in di.dart resolver

**Review by mkyq — CHANGES_REQUESTED** [review]

**Comment by mkyq** [comment]
cc @RafiaChy

File: lib/view_model/dashboard/balance_view_model.dart (hunk 1 above)
Comment (RafiaChy) [reply lib/view_model/dashboard/balance_view_model.dart:70]: Okay Sir

File: lib/view_model/dashboard/balance_view_model.dart (hunk 2 above)
Comment (RafiaChy) [reply lib/view_model/dashboard/balance_view_model.dart:147]: Noted

File: lib/src/screens/dashboard/widgets/balance_page.dart (hunk 4 above)
Comment (RafiaChy) [reply lib/src/screens/dashboard/widgets/balance_page.dart:16]: Okay, Sir

File: lib/di.dart [hunk 5]
```diff
@@ -302,7 +304,12 @@ Future setup(
               onAuthenticationFinished: onAuthFinished,
               closable: closable ?? false));
 
+  getIt.registerFactory(() => 
+   BalancePage(dashboardViewModel: getIt.get<DashboardViewModel>(), 
+   settingsStore: getIt.get<SettingsStore>()));
+
   getIt.registerFactory<DashboardPage>(() => DashboardPage(
+      balancePage: BalancePage(dashboardViewModel: getIt.get<DashboardViewModel>(), settingsStore: getIt.get<SettingsStore>()),
```
Comment (mkyq) [inline lib/di.dart:312]: Inject here BalancePage by using getIt (getIt.get<BalancePage>())

File: lib/di.dart [hunk 6]
```diff
@@ -248,6 +249,7 @@ Future setup(
       fiatConvertationStore: getIt.get<FiatConversionStore>()));
 
   getIt.registerFactory(() => DashboardViewModel(
+
```
Comment (mkyq) [inline lib/di.dart:252]: Is that single empty line is necessary ? if it's not necessary please remove it

**Review by mkyq — CHANGES_REQUESTED** [review]
cc @RafiaChy

File: lib/di.dart (hunk 6 above)
Comment (RafiaChy) [reply lib/di.dart:252]: Done

**Review by mkyq — APPROVED** [review]

**Pull request merged**